app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -147,11 +147,21 @@
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_marginTop="16dp"
-            android:text="@string/terms_and_conditions_acceptance_statement"
-            app:layout_constraintEnd_toEndOf="@+id/tiConfirmPassword"
             app:layout_constraintStart_toStartOf="@+id/tiConfirmPassword"
             app:layout_constraintTop_toBottomOf="@+id/cbMentee" />
 
+        <androidx.appcompat.widget.AppCompatTextView
+            android:id="@+id/cbTOC"
```

Comment:
Can you change this id to `tvTC`, this is how we've been naming ids, `tv` stands for `textview`, while `cb` stands for `checkbox`. TC is for Terms and Conditions.